### PR TITLE
TRITON-2304 New image server names

### PR DESCRIPTION
**Pull request by bahamat**
Note: This is only changes to comments, so no version bump. Also will not be re-published to NPM at this time.

### CODE REVIEW

**Review by danmcd — APPROVED** [review]
Good timing (after branch).  Is this part one of a full TRITON-2304 fix?  If so, you may wish to annotate that, e.g.:

    TRITON-2304 New image server names (comments first)
    
Just a suggestion, NOT a blocker by any means.

**Comment by bahamat** [comment]
In this repo the only references are comments. Others won't be restricted to just comments. See tritondatacenter/sdc-imgapi-cli#7

File: CHANGES.md [hunk 1]
```diff
@@ -8,7 +8,11 @@
 
 ## Not yet released
 
-(nothing yet)
+- TRITON-2304 New image server names
+
+## 13.0.5
+
+- TOOLS-2525 Everything needs to stop cloning with git:// URLs
```
Comment (danmcd) [inline CHANGES.md:15]: Not a blocker, but was this one just missed earlier?

**Review by danmcd — APPROVED** [review]
One non-blocking question...

**Pull request merged**